### DISCUSSION TITLE
Custom location on Github organization

### DISCUSSION THREAD

**Discussion by morphlings** [community/community#19856] [Projects and Issues]
How to set custom location on my github organization? Iirc the old github still supporting this feature instead dropdown list.. The current github just provided limited country and registered city related it.

**Comment by sinkaroid** [comment]
I think it's not related issues and working as intended, the https://api.github.com still accepting your arbitrary requests

```sh
curl -H "Authorization: bearer your_token" -d '{"location":"your_desired_value"}' https://api.github.com/orgs/your_orgs
```

Reference: https://stackoverflow.com/questions/68970902/custom-location-text-for-github-organization

**Reply by morphlings** [reply]
I have been follow this step and result me to Bad credentials

**Reply by sinkaroid** [reply]
We don't know the credentials that you have using it, The github token should start with the pattern `ghp_xxx`, does it match your auth?  
If you doubt, try to regenerate again

**Reply by morphlings** [reply]
I don't know something wrong with my previous token, created the new one and it's working like a charm, thankyou
